### DISCUSSION TITLE
Add ROC metric to AnomalyModule

### DISCUSSION THREAD

**Discussion by dreaquil** [open-edge-platform/anomalib#186] [Feature Requests] [answered]
**Is your feature request related to a problem? Please describe.**
Currently only AUROC and F1 for image and pixel are measured. I think it would be useful to (optionally) include the full ROC.

**Describe the solution you'd like**
 It would be useful to have the ROC metric included as standard or optional.

**Note**
ROC is a bit problematic as a metric because it is currently not possible to simply LightningModule.log or log_dict it.

**Comment by ashwinvaidya17** [comment, marked as answer]
This PR (https://github.com/openvinotoolkit/anomalib/pull/429) adds plots for ROC curves. It should address your need
